src/landing_page/src/components/outputs/outputs.js

```diff
@@ -72,14 +72,18 @@ function Output(props) {
   let standards = output?.metadata || [];
   let distributions = props?.output?.distribution || [];
   let hosts = Array.isArray(distributions) && distributions.length > 0 ? distributions.map((obj) => { return obj?.host }) : [];
-  let licenses = Array.isArray(distributions) && distributions.length > 0 ? distributions.map((obj) => { return obj?.license }) : [];
+  let licenses = Array.isArray(distributions) && distributions.length > 0
```

Comment:
Sorry about all the noise that was added by my prettier formatting.

This is the main change that I made. Essentially I made an update to filter out "undefined" from the array.